docs/step-by-step/kubeapps-on-tkg/step-3.md

```diff
@@ -1,138 +1,115 @@
 
-> **NOTE**: To obtain the most current list of repositories, log in to the registry using the `oras` tool and execute the command:
+> **NOTE**: To obtain the most current list of repositories, log in to the registry using the `oras` tool and execute the command below:
 >
 > ```bash
 > oras pull registry.pivotal.io/tac-for-tanzu-advanced/index:latest -a && cat asset-index.json | jq -r '.charts | map(.name) | join(",")'
 > ```
 
-## 3.X - RBAC in Application Repositories
+At the end of this step, the Kubeapps installation is configured with one or more application repositories. The next step is to [start using Kubeapps](./step-4.md).
 
-A Kubeapps `AppRepository` can be created by anyone with the required RBAC for that namespace.
+## Appendix: Role-Based Access Control (RBAC) in Application Repositories
 
-If you have cluster-wide RBAC for creating AppRepositories, you can still create an `AppRepository` whose charts will be available to users in all namespaces by selecting `All Namespaces` when creating the repository.
+A Kubeapps application repository can be created by anyone with the required RBAC privileges for that namespace.
 
-To grant a user (e.g., `USERNAME`) permissions to create `AppRepositories` objects in a specific namespace (e.g., `CUSTOM-NAMESPACE`), you must create `RoleBinding` objects associating `apprepositories-read` and `apprepositories-write` as follows.
+Users with cluster-wide RBAC privileges for creating application repositories  can still create an application repository whose charts will be available to users in all namespaces by selecting `All Namespaces` when creating the repository.
 
-> **TIP**: if you only require granting users the with permissions to deploy charts from `AppRepositories` in a specific namespace, create only the `apprepositories-read` `RoleBinding` object.
+To grant a user permission to create `AppRepository` objects in a specific namespace,  create `RoleBinding` objects associating `apprepositories-read` and `apprepositories-write` roles with that user, as shown below. Replace the `USERNAME` and `CUSTOM-NAMESPACE` placeholders with the corresponding username and namespace name.
+
+> **TIP**: If users only require permissions to deploy charts from application repositories in a specific namespace, it is sufficient to create only the `apprepositories-read` `RoleBinding` object.
 
 ```bash
-# Grant the user <USERNAME> read apprepositories permissions on the <CUSTOM-NAMESPACE> namespace
-kubectl -n <CUSTOM-NAMESPACE> create rolebinding <USERNAME>-apprepositories-read \
-    --user <USERNAME>
+# Grant the user USERNAME read apprepositories permissions on the CUSTOM-NAMESPACE namespace
+kubectl -n CUSTOM-NAMESPACE create rolebinding USERNAME-apprepositories-read \
+    --user USERNAME
     --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-read
 
     --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-write
```

Comment:
Should this line be here or is it a mistake? It is repeated again below.

Reply:
Ah, yep, we should have a code fragment for: i) apprepositories-read and ii) apprepositories-write.

It has been extracted from: https://github.com/kubeapps/kubeapps/blob/master/docs/user/access-control.md#assigning-kubeapps-user-roles

Perhaps we should point to that part directly (fine grained config) and here just create a `cluster-admin` cluster-role (all permission)

The latter is sth like:
```

# kubectl apply -f kubeapps-rbac.yaml

kind: ClusterRoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: kubeapps-operator
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
  - apiGroup: rbac.authorization.k8s.io
    kind: User
    name: agamez@vmware.com

```